Library/Homebrew/utils/pypi.rb

```diff
@@ -284,17 +285,20 @@ def self.update_python_resources!(formula, version: nil, package_name: nil, extr
     true
   end
 
-  def self.json_to_packages(json_tree, main_package, exclude_packages)
-    return [] if json_tree.blank?
+  def self.normalize_python_package(name)
+    name.gsub(/[-_.]+/, "-").downcase
+  end
```

Comment:
This normalization is defined in <https://packaging.python.org/en/latest/specifications/name-normalization/#name-normalization>; I'll add that as a comment.